database/migrations/2022_07_08_145847_create_spam_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        Schema::create('spam', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('user_id')->nullable();
+            $table->foreign('user_id')->references('id')->on('users')->cascadeOnDelete();
+            $table->unsignedInteger('thread_id')->nullable();
+            $table->foreign('thread_id')->references('id')->on('threads')->cascadeOnDelete();
```

Comment:
I'm wondering if we should make this polymorphic somehow? We should also be able to report spam replies eventually.

Reply:
that makes sense 🤔

Reply:
resolved here https://github.com/laravelio/laravel.io/pull/886/commits/5cc272cfa8ca49d567badbed15319f56746ed820